js/weather.js

```diff
@@ -1,35 +1,91 @@
 $(function () {
+  let selectedBox = 'current';
+  let apiData = {};
+
+  $('.toggle-buttons').click(function (e) {
+    // prevent checked box to be unchecked on click
+    if (!$(this).is(':checked')) {
+      e.preventDefault;
+      return false;
+    }
+    $(this).siblings().attr('checked', false); // uncheck checked siblings
+    selectedBox = $(this)[0].value;
+
+    if (Object.keys(apiData).length) {
+      displayResults();
+    }
+  });
+
   // React to hitting enter in the text box instead of clicking submit.
-  $("#search-weather").submit(function(e) {
+  $('#search-weather').submit(function (e) {
```

Comment:
Nice. I assume this may be why you switched to the checkbox concept... so it would know which to use if user submits with enter. 

Maybe for the enter option, one button could be colored to indicate it's the default if it's not clicked (hits enter instead of clicking button). The color and output would let the user know which was chosen for them. I see benefits of the other approach too if a user wanted to check one box then be able to try multiple locations without more checking.

Reply:
In pull requests, be sure to add a reply to explain how you resolved or to make a case for doing something differently than suggested. If you're sure it's resolved because it's a little change or from discussion with the reviewer, then you can click `Resolve conversation`.

Reply:
Gotcha, so since the current project that I am working on is more functionality centric, I believed that implementing radio buttons, without styling, was the appropriate process for the time being. Then once I move onto project-3, I would be able to add the proposed styling.